src/Auth/ResetsPasswords.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Corcel\Auth;
+
+use Corcel\Password\PasswordService;
+use Auth;
+
+trait ResetsPasswords {
+	/**
+     * Reset the given user's password.
+     *
+     * @param  \Illuminate\Contracts\Auth\CanResetPassword  $user
+     * @param  string  $password
+     * @return void
+     */
+    protected function resetPassword($user, $password)
+    {
+    	$passwordService = new PasswordService;
+
+        $user->user_pass = $passwordService->wp_hash_password($password);
+
+        $user->save();
+        
+        Auth::guard($this->getGuard())->login($user);
```

Comment:
Hmmm, [I just took this code from Laravel](https://github.com/laravel/framework/blob/cf01460ff629cb80dfff3831c127a0b31edb9964/src/Illuminate/Foundation/Auth/ResetsPasswords.php#L214-L221), I only changed how the password is encrypted ...
If someone is not using Laravel, there's no need for him to use this trait.